### PR TITLE
fixed password mask

### PR DESCRIPTION
**Pull request by kimw**
changed password mask from star mark `*` to bullet marker `∙`

### CODE REVIEW

File: shadowsocks-csharp/View/ConfigForm.Designer.cs [hunk 1]
```diff
@@ -174,10 +174,10 @@ private void InitializeComponent()
             this.PasswordTextBox.Location = new System.Drawing.Point(83, 60);
             this.PasswordTextBox.MaxLength = 256;
             this.PasswordTextBox.Name = "PasswordTextBox";
-            this.PasswordTextBox.PasswordChar = '*';
             this.PasswordTextBox.Size = new System.Drawing.Size(160, 21);
             this.PasswordTextBox.TabIndex = 2;
             this.PasswordTextBox.WordWrap = false;
+	    this.PasswordTextBox.UseSystemPasswordChar = true;
```
Comment (wongsyrone) [inline shadowsocks-csharp/View/ConfigForm.Designer.cs]: wrong indent

**Comment by kimw** [comment]
Fixed in ba7bfc4.
@wongsyrone Thanks for your checking.

File: shadowsocks-csharp/View/ConfigForm.Designer.cs (hunk 1 above)
Comment (kimw) [inline shadowsocks-csharp/View/ConfigForm.Designer.cs]: Fixed in ba7bfc4.
@wongsyrone Thanks for your checking.

**Comment by GangZhuo** [comment]
Thanks

**Pull request merged**